jwst/ramp_fitting/ramp_fit.py

```diff
@@ -219,24 +243,22 @@ def ols_ramp_fit(model, buffsize, save_opt, readnoise_model, gain_model,
 
     opt_res = utils.OptRes(n_int, imshape, max_seg, nreads, save_opt)
 
-    # Get Pixel DQ array from input file. The incoming RampModel has uint8
-    #   PIXELDQ, but ramp fitting will update this array here by flagging
-    #   the 2D PIXELDQ locations where the ramp data has been previously
-    #   flagged as jump-detected or saturated. These additional bit values
-    #   require this local variable to be uint16, and it will be used as the
-    #   (uint16) PIXELDQ in the outgoing ImageModel.
-    pixeldq = model.pixeldq.copy()
-
-    # calculate number of (contiguous) rows per data section
+    # Calculate number of (contiguous) rows per data section
     nrows = calc_nrows(model, buffsize, cubeshape, nreads)
 
     # Get readnoise array for calculation of variance of noiseless ramps, and
     #   gain array in case optimal weighting is to be done
     readnoise_2d, gain_2d = utils.get_ref_subs(model, readnoise_model,
                                                gain_model, nframes)
 
-    # Flag any bad pixels in the gain
-    pixeldq = utils.reset_bad_gain( pixeldq, gain_2d )
+    # Get Pixel DQ array from input file. The incoming RampModel has uint8
```

Comment:
The pixeldq in RampModel is uint32, so this should be OK to start with.  (It's the groupdq that's uint8 in RampModel.)